awesome_collections.md

```diff
@@ -0,0 +1,13 @@
+# Awesome Hugging Face Collections
+
+<p align="center"><em>🎃 Note as part of [Hacktoberfest 2023](https://hacktoberfest.com/), we're looking for contributions to this page! If you have a collection you'd like to add, please submit a PR! See this issue for more context! #TODO add issue link 🎃</em></p>
+
+[Hugging Face Collections](https://huggingface.co/docs/hub/collections) provide a way of curating repositories from the Hub (Models, Datasets, Spaces and Papers) on a dedicated page. This page collects some examples of awesome collections! 
```

Comment:
```suggestion
[Hugging Face Collections](https://huggingface.co/docs/hub/collections) provide a way of curating repositories from the Hub (models, datasets, Spaces and papers) on a dedicated page. This page collects some examples of awesome collections! 
```

Usually only `Spaces` has a capital letter in the docs (AFAIU, to brand it more as a product?)